cmd/agent/app/configmanager/grpc/manager_test.go

```diff
@@ -38,7 +38,7 @@ func TestSamplingManager_GetSamplingStrategy(t *testing.T) {
 		api_v2.RegisterSamplingManagerServer(s, &mockSamplingHandler{})
 	})
 	conn, err := grpc.NewClient(addr.String(), grpc.WithTransportCredentials(insecure.NewCredentials()))
-	defer close(t, conn)
+	defer closeTestGrpcConn(t, conn)
```

Comment:
This function is superfluous, can inline as cleanup 
```suggestion
	t.Cleanup(func() { require.NoError(t, conn.Close()) })
```